firefox_puppeteer/ui/toolbars.py

```diff
@@ -446,3 +446,20 @@ def verifier(self):
         :returns: Reference to the identity-popup content verifier.
         """
         return self.marionette.find_element(By.ID, 'identity-popup-content-verifier')
+
+    def close(self, force=False):
+        """Closes the identity popup by hitting the escape key.
+
+        :param force: Optional, flag that, if true, will force close the popup.
```

Comment:
And it also misses to mention the default value. See other libs how those are doing it.